modules/ROOT/pages/maven-preferences-reference.adoc

```diff
@@ -39,11 +49,57 @@ Selecting this option requires you to define your custom path to your Maven loca
 | Empty
 
 | Base command line for builds
-| The Maven command to execute every time you run your Mule project in your local machine.
+| The Maven command to execute every time you build or run a Mule project in your local machine.
 | mvn clean package -nsu -DskipMunitTests
 
 |===
 
+== Configure Global Settings
+
+Studio manages the servers used in the global settings and you can configure Studio to use an external Maven installation. By doing so, Studio automatically detects and populates the IDs from the `settings.xml` file stored in `${maven.installation.home}/conf/settings.xml`.
+
+image::maven-global-settings.png[“*Global Settings* option highlighted in the Studio setting preferences window.”]
+
+If there is no file on this path, you can create a new `settings.xml` file in another location:
+
+. Select *Override global settings path*.
+. Click *Browse*.
+. Select your `settings.xml` file and click *Open*.
++ The IDs are listed.
+. Click *Apply and Close*.
+
+== Configure Update Policies
+
+You can configure Maven update policies depending on your needs.
+
+image::maven-update-policies.png[“*Update Policies* option highlighted in the Studio setting preferences window.”]
+
+See https://maven.apache.org/settings.html#repositories[Update Policies^] for additional information.
+
+== Configure User Settings
+
+Studio manages the servers for downloading dependencies or plugins, defines the credentials of such servers when needed, and automatically detects and populates the IDs from the `settings.xml` file located in `${user.home}/.m2/settings.xml`.
+
+image::maven-user-settings.png[“*User Settings* option highlighted in the Studio setting preferences window.”]
+
+If there is no file on this path, you can create a new `settings.xml` file in another location:
+
+. Select *Override user settings path*.
+. Click *Browse*.
+. Select your `settings.xml` file and click *Open*.
++ The IDs are listed.
+. Click *Apply and Close*.
+
+=== Add a Server
+
+To add a new server to your `settings.xml` file: 
+
+. In the *User Settings* window, click *Add*.
+. Enter the server ID, and your username and password.
+. Click *OK*.
+
+image::maven-add-new-server.png[“*Add server* option highlighted in the Studio setting preferences window.”]
+
 == See Also
 
-* xref:add-modules-in-studio-to.adoc[To Add Modules to Your Project]
+* xref:add-modules-in-studio-to.adoc[]
```

Comment:
Will it get the title of the referenced .adoc as link text?

Reply:
Yes! The build automagically gets the title name and fills the emptiness hahhaa
Thanks!